### PR TITLE
Fix to build afl-clang-lto.

### PR DESCRIPTION
**Pull request by kobrineli**

### CODE REVIEW

File: docker/ubuntu20.04-sydr-fuzz/Dockerfile [hunk 1]
```diff
@@ -49,12 +60,13 @@ RUN git clone --depth=1 https://github.com/vanhauser-thc/afl-cov /afl-cov && cd
...
 
 ENV LLVM_CONFIG=llvm-config
+ENV LD_LIBRARY_PATH="$(llvm-config --libdir)${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}"
 ENV AFL_I_DONT_CARE_ABOUT_MISSING_CRASHES=1
 ENV IS_DOCKER="1"
 
 RUN git clone https://github.com/AFLplusplus/AFLplusplus.git && cd AFLplusplus && \
     git checkout 40947508037b874020c8dd1251359fecaab04b9d && \
-    export CC=gcc-10 && export CXX=g++-10 && make clean && \
+    export CC=clang && export CXX=clang++ && make clean && \
```
Comment (SweetVishnya) [inline docker/ubuntu20.04-sydr-fuzz/Dockerfile:69]: export LD_LIBRARY_PATH here

File: docker/ubuntu20.04-sydr-fuzz/Dockerfile [hunk 2]
```diff
@@ -31,14 +31,25 @@ RUN wget https://github.com/ninja-build/ninja/releases/download/v1.10.2/ninja-li
...
-                  -GNinja && \
-    CMAKE_BUILD_PARALLEL_LEVEL=$(nproc) cmake --build . && cmake --build . && \
+    cmake \
+	-DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_C_COMPILER=clang-11 \
+	-DCMAKE_CXX_COMPILER=clang++-11 \
+	-DCLANG_INCLUDE_DOCS="OFF" \
+	-DCMAKE_BUILD_TYPE=Release \
+	-DLLVM_BINUTILS_INCDIR=/usr/include/ \
+	-DLLVM_BUILD_LLVM_DYLIB="ON" \
+	-DLLVM_ENABLE_BINDINGS="OFF" \
```
Comment (SweetVishnya) [inline docker/ubuntu20.04-sydr-fuzz/Dockerfile:41]: remove this

File: docker/ubuntu20.04-sydr-fuzz/Dockerfile [hunk 3]
```diff
@@ -31,14 +31,25 @@ RUN wget https://github.com/ninja-build/ninja/releases/download/v1.10.2/ninja-li
...
+	-DLLVM_ENABLE_ZLIB=ON \
+	-DLLVM_ENABLE_PROJECTS='clang;compiler-rt;libcxx;libcxxabi;libunwind;lld;clang-tools-extra' \
```
Comment (SweetVishnya) [inline docker/ubuntu20.04-sydr-fuzz/Dockerfile:43]: RUNTIMES

File: docker/ubuntu20.04-sydr-fuzz/Dockerfile [hunk 4]
```diff
@@ -31,14 +31,25 @@ RUN wget https://github.com/ninja-build/ninja/releases/download/v1.10.2/ninja-li
...
+	-DLLVM_ENABLE_WARNINGS="OFF" \
+	-DLLVM_INCLUDE_BENCHMARKS="OFF" \
+	-DLLVM_INCLUDE_DOCS="OFF" \
+	-DLLVM_INCLUDE_EXAMPLES="OFF" \
+	-DLLVM_INCLUDE_TESTS="OFF" \
+	-DLLVM_LINK_LLVM_DYLIB="ON" \
+	-DLLVM_TARGETS_TO_BUILD="host" \
```
Comment (SweetVishnya) [inline docker/ubuntu20.04-sydr-fuzz/Dockerfile:50]: remove this

**Review by SweetVishnya — CHANGES_REQUESTED** [review]

File: docker/ubuntu20.04-sydr-fuzz/Dockerfile (hunk 4 above)
Comment (kobrineli) [reply docker/ubuntu20.04-sydr-fuzz/Dockerfile]: done

File: docker/ubuntu20.04-sydr-fuzz/Dockerfile (hunk 3 above)
Comment (kobrineli) [reply docker/ubuntu20.04-sydr-fuzz/Dockerfile]: done

File: docker/ubuntu20.04-sydr-fuzz/Dockerfile (hunk 1 above)
Comment (kobrineli) [reply docker/ubuntu20.04-sydr-fuzz/Dockerfile]: done

File: docker/ubuntu20.04-sydr-fuzz/Dockerfile [hunk 5]
```diff
@@ -31,14 +31,24 @@ RUN wget https://github.com/ninja-build/ninja/releases/download/v1.10.2/ninja-li
...
-    CMAKE_BUILD_PARALLEL_LEVEL=$(nproc) cmake --build . && cmake --build . && \
+    cmake \
+	-DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_C_COMPILER=clang-11 \
+	-DCMAKE_CXX_COMPILER=clang++-11 \
+	-DCLANG_INCLUDE_DOCS="OFF" \
+	-DCMAKE_BUILD_TYPE=Release \
+	-DLLVM_BINUTILS_INCDIR=/usr/include/ \
+	-DLLVM_BUILD_LLVM_DYLIB="ON" \
+	-DLLVM_ENABLE_ZLIB=ON \
+        -DLLVM_ENABLE_RUNTIMES="libcxx;libcxxabi;libunwind" \
```
Comment (SweetVishnya) [inline docker/ubuntu20.04-sydr-fuzz/Dockerfile:42]: extra spaces

Comment (kobrineli) [reply docker/ubuntu20.04-sydr-fuzz/Dockerfile:42]: done

**Review by SweetVishnya — APPROVED** [review]

**Pull request merged**